README.md

```diff
@@ -21,6 +21,7 @@ use ZipkinTracer::RackHandler, config
 * `:service_name` **REQUIRED** - the name of the service being traced. There are two ways to configure this value. Either write the service name in the config file or set the "DOMAIN" environment variable (e.g. 'test-service.example.com' or 'test-service'). The environment variable takes precedence over the config file value.
 * `:sample_rate` (default: 0.1) - the ratio of requests to sample, from 0 to 1
 * `:sampled_as_boolean` - When set to true (default but deprecrated), it uses true/false for the `X-B3-Sampled` header. When set to false uses 1/0 which is preferred.
+* `:check_routes` - When set to true, only routable requests are sampled.
```

Comment:
explictily mention that it's false by default?

Reply:
added b86b191